recipes/glue-qt/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "glue-qt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/glue-qt-{{ version }}.tar.gz
+  sha256: 26a7f10e8bb58426512921355a0b9078698170fcc66504b2174969e1d1073d42
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - glue = glue_qt.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools-scm
+    - pip
+  run:
+    - python
+    - glue-core
+    - numpy >=1.17
+    - matplotlib-base >=3.2
+    - scipy >=1.1
+    - echo >=0.6
+    - astropy >=4.0
+    - setuptools >=30.3.0
+    - qtpy >=1.9
+    - ipython >=4.0
+    - ipykernel >=4.0,!=5.0.0,!=5.1.0
+    - qtconsole >=4.3,!=5.4.2
+    - pvextractor >=0.2
+    - importlib-resources >=1.3  # [py<39]
+    - importlib-metadata >=3.6  # [py<310]
```

Comment:
We have empty packages for these on modern Python. That way you can drop the selector and make this noarch.

Reply:
Thanks! Actually I just realised I need to add in python.app as a dependency which is OS-X only so can't actually specify noarch - but I've removed the selectors here.